packages/dd-trace/src/plugins/util/test.js

```diff
@@ -173,6 +173,13 @@ function getTestParametersString (parametersByTestName, testName) {
   }
 }
 
+function getTestTypeFromFramework (testFramework) {
+  if (testFramework.includes('playwright') || testFramework.includes('cypress')) {
```

Comment:
let's simply do. Those frameworks should be fixed
```suggestion
  if (testFramework === 'playwright' || testFramework === 'cypress') {
```